### PR TITLE
MLPAB-1340 Add flow for 2 witnesses

### PR DESCRIPTION
**Pull request by hawx**

### CODE REVIEW

File: internal/page/donor/witnessing_as_independent_witness_test.go [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+		DonorIdentityUserData:   identity.UserData{OK: true, Provider: identity.OneLogin},
+		IndependentWitness:      actor.IndependentWitness{},
+		IndependentWitnessCodes: page.WitnessCodes{{Code: "1234", Created: now}},
+	})
+	resp := w.Result()
+
+	assert.Nil(t, err)
+	assert.Equal(t, http.StatusFound, resp.StatusCode)
+	assert.Equal(t, page.Paths.WitnessingAsCertificateProvider.Format("lpa-id"), resp.Header.Get("Location"))
+}
```
Comment (acsauk) [inline internal/page/donor/witnessing_as_independent_witness_test.go:148]: Is there anything different being asserted here besides a witness being set on the LPA passed to the handler?

**Review by acsauk — APPROVED** [review]
One Q on a test but I may just be missing something subtle.

:shipit:

File: internal/page/donor/witnessing_as_independent_witness_test.go (hunk 1 above)
Comment (hawx) [reply internal/page/donor/witnessing_as_independent_witness_test.go]: you're right, it was some bad copypasta

**Review by acsauk — APPROVED** [review]
:shipit:

**Pull request merged**